napari/utils/action_manager.py

```diff
@@ -93,13 +93,9 @@ def _debug(self, val):
 
     def _validate_action_name(self, name):
         if len(name.split(':')) != 2:
-            raise ValueError(
-                trans._(
-                    'Action names need to be in the form `package:name`, got {name!r}',
-                    name=name,
-                    deferred=True,
-                )
-            )
+            return False  # this action comes from app_model
+        else:
+            return True
```

Comment:
Maybe annotate the return type here now as it changed?

Also I'm not sure this heuristic is correct as you could still have an action name that passes here but it still came from app-model (app-model would support `package:name` as well). I don't know if this matters.

Reply:
I am also not familiar with the name convention and it seems we are ditching this requirement? @kne42 how do you feel about the logic here?

Reply:
i think the check for if an action comes from app-model vs the action manager should happen elsewhere, as it seems that we are simply not executing the logic at all if it's from app-model

Reply:
+1 to @kne42's comment - I thought I wrote something like that but it must not have saved. The way this validate function has changed it causes several functions to silently no-op (fail, really). Maybe this is not a big deal if it's just a transitional thing until action_manager is removed? I would resist merging to main in this state though unless well documented somewhere.